index.bs

```diff
@@ -8736,18 +8737,20 @@ create objects that implement the interface.
 Multiple [{{NamedConstructor}}] extended
 attributes may appear on a given interface.
 
-The [{{NamedConstructor}}]
-extended attribute must either
+The [{{NamedConstructor}}] extended attribute must either
 [=takes an identifier|take an identifier=] or
 [=takes a named argument list|take a named argument list=].
-The first form, <code>[NamedConstructor=<emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t>]</code>, has the same meaning as
-using an empty argument list, <code>[NamedConstructor=<emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t>()]</code>.  For each
-[{{NamedConstructor}}] extended attribute
-on the interface, there will be a way to construct an object that implements
-the interface by passing the specified arguments to the constructor function
+The <emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t> that occurs directly after the
+“<emu-t>=</emu-t>” is the [{{NamedConstructor}}]'s <dfn lt="NamedConstructor identifier">identifier</dfn>.
+The first form, <code>[NamedConstructor=<emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t>]</code>,
+has the same meaning as using an empty argument list,
+<code>[NamedConstructor=<emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t>()]</code>.
+For each [{{NamedConstructor}}] extended attribute on the interface,
+there will be a way to construct an object that
+implements the interface by passing the specified arguments to the constructor function
 that is the value of the aforementioned property.
 
-The identifier used for the named constructor must not
+The [=NamedConstructor identifier|identifier=] used for the named constructor must not
```

Comment:
I could go either way. We don't have separate #x-identifier concepts for everything else. On the other hand, the "after the = sign" thing down below does sure seem out of place, so moving it up here is not a bad idea.